scripts/get-salt-version.sh

```diff
@@ -30,7 +30,7 @@ compare_version () {
 BASE_NAME=$1
 STACK_VERSION=$2
 
-SALT_VERSION=3001.8
+SALT_VERSION=3004.2
```

Comment:
This change affects only FreeIPA images. Is it intended?

Reply:
In general: what's the final verdict? Is this really required for FIPS mode?

Reply:
There was no final verdict afaik, but fips_mode config above is not supported in 3001.8.

Reply:
tested with 3001.8 again, restored